backend/imap/attachments.go

```diff
@@ -53,7 +53,12 @@ func (b *Messages) ReadAttachment(user, id string) (att *backend.Attachment, out
 		return
 	}
 
-	att, r := parseAttachment(data.GetBody("BODY["+partId+"]"))
+        var bodySectionName *imap.BodySectionName
```

Comment:
Please use tabs for indentation! (you can `go fmt` your source code to fix it btw)